src/generalizedlinearmixedmodel.jl

```diff
@@ -445,6 +445,8 @@ end
 getθ(m::GeneralizedLinearMixedModel)  = copy(m.θ)
 getθ!(v::AbstractVector{T}, m::GeneralizedLinearMixedModel{T}) where {T} = copyto!(v, m.θ)
 
+StatsBase.islinear(m::GeneralizedLinearMixedModel) = isa(GLM.Link, GLM.IdentityLink)
```

Comment:
The documentation for `StatsBase,islinear` is vague.  I am interpreting it to mean that the linear predictor is the conditional mean.